package-lock.json

```diff
@@ -8,26 +8,27 @@
       "name": "roommate-organizer",
       "version": "0.1.0",
       "dependencies": {
-        "@prisma/client": "^5.9.1",
+        "@prisma/client": "^5.10.2",
         "@supabase/auth-helpers-nextjs": "^0.9.0",
         "@supabase/ssr": "^0.1.0",
         "@supabase/supabase-js": "^2.39.7",
         "dotenv": "^16.4.1",
         "next": "14.1.0",
         "next-swagger-doc": "^0.4.0",
         "postgres": "^3.4.3",
-        "prisma": "^5.9.1",
         "react": "^18",
         "react-dom": "^18",
         "react-icons": "^5.0.1",
         "react-loading-skeleton": "^3.4.0",
+        "react-router-dom": "^6.22.2",
```

Comment:
We shouldn't need the `react-router-dom package`, Next.js provides routing as their main feature – let's remove that package